airframe-json/src/main/scala/wvlet/airframe/json/JSON.scala

```diff
@@ -125,15 +133,22 @@ object JSON extends LogSupport {
       }
     }
   }
-  final case class JSONArray(v: IndexedSeq[JSONValue]) extends JSONValue {
+  final case class JSONArray(v: Seq[JSONValue]) extends JSONValue {
```

Comment:
For supporting `apply(i: Int)`, IndexSeq or Vector is usually better, but it might be rare to access JSON array elements with an index, so this change should be ok.